spectrum-network/src/peer_conn_handler.rs

```diff
@@ -259,26 +360,33 @@ impl ConnectionHandler for PeerConnHandler {
                     ProtocolState::Accepting {
```

Comment:
Is it possible for state to have 'accepting' status on inject_fully_negotiated_outbound call? Looks like the only acceptable state is 'opening'

Reply:
Please take a look into "docs/Networking.md" to get an idea of how protocol negotiation is done.

Reply:
<img width="146" alt="image" src="https://user-images.githubusercontent.com/33941049/201325320-c61c850b-9fef-4657-b1ae-a135cbe7360f.png">

There is no "Accepting" state

Reply:
Take a look a the left side

Reply:
I'm talking about outbound process

Reply:
Exactly, each side performs inbound and outbound stream negotiation. Please look carefully.